sherpa-onnx/csrc/online-lm-config.cc

```diff
@@ -34,7 +36,8 @@ std::string OnlineLMConfig::ToString() const {
 
   os << "OnlineLMConfig(";
   os << "model=\"" << model << "\", ";
-  os << "scale=" << scale << ")";
+  os << "scale=" << scale << ", ";
+  os << "shallow_fusion=\"" << (shallow_fusion ? "True" : "False") << "\")";
```

Comment:
```suggestion
  os << "shallow_fusion=" << (shallow_fusion ? "True" : "False") << ")";
```

We use `""` only for string variables.